portal/src/locale-data/en.json

```diff
@@ -232,6 +233,13 @@
   "PasswordsScreen.forgot-password.sms.content.label": "Content",
   "PasswordsScreen.forgot-password.time-to-invalid-reset-code.label": "Time to invalid reset code (seconds)",
 
+  "PasswordlessAuthenticatorScreen.title": "Passwordless Authenticator",
+  "PasswordlessAuthenticatorScreen.email.label": "Email",
+  "PasswordlessAuthenticatorScreen.email.styled-content.label": "Styled Content",
```

Comment:
I guess we should use the terms that are more popular and easier to understand.

```
"html-email-label": "HTML Email"
"plaintext-email-label": "Plaintext Email"
"sms-body-label": "SMS Body"
```

Reply:
Fixed